src/main/java/com/github/dockerunit/discovery/consul/ConsulDescriptor.java

```diff
@@ -53,13 +55,14 @@ public CreateContainerCmd setup(CreateContainerCmd cmd) {
 
         return cmd.withHostConfig(hc)
                 .withExposedPorts(ports)
+                .withImage(System.getProperty("consul.image", CONSUL_DEFAULT_IMAGE))
```

Comment:
wdyt to have both constants or both strings?

Reply:
Agree. It would be nice to be consistent with the way it is done here https://github.com/dockerunit/dockerunit-consul/blob/master/src/main/java/com/github/dockerunit/discovery/consul/ConsulDiscoveryConfig.java#L10